### PR TITLE
feat: Support optional ClassVar serialization

### LINKED ISSUE #296: `ClassVar`s don't get serialized?
Example:
```from typing import ClassVar
from serde.se import serialize, to_dict

@serialize
class Foo:
    value: ClassVar[str] = "abc"

print(to_dict(Foo()))  # Prints {}
```

### PR DESCRIPTION
**Pull request by yukinarit**
```
@serialize(serialize_class_var=True)
class Foo:
    v: typing.ClassVar[int] = 10
```

Closes #296

### CODE REVIEW

File: serde/compat.py [hunk 1]
```diff
@@ -124,7 +137,7 @@ def get_origin(typ: Type[Any]) -> Optional[Any]:
     try:
         return typing.get_origin(typ) or get_np_origin(typ)  # python>=3.8 typing module has get_origin.
     except AttributeError:
-        return typing_inspect.get_origin(typ) or get_np_origin(typ)
+        return typing_extensions.get_origin(typ) or get_np_origin(typ)
```
Comment (yukinarit) [inline serde/compat.py:140]: Because `typing_inspect.get_origin(ClassVar[int])` returns `None` instead of `ClassVar`.

**Comment by yukinarit** [comment]
@AustinScola
Would you please take a look?

File: examples/class_var.py [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
 
 
-def main():
-    f = Foo(i=10, j=20)
+def main() -> None:
+    f = Foo()
     print(f"Into Json: {to_json(f)}")
 
-    s = '{"i": 10, "j": 20}'
+    s = '{"v": 10}'
```
Comment (AustinScola) [inline examples/class_var.py:26]: Do you want the key to be `v` here?

**Review by AustinScola — APPROVED** [review]
Awesome, thank you for such a quick addition! I really appreciate the work you put into `pyserde` :)

**Comment by yukinarit** [comment]
@AustinScola Thanks!

**Pull request merged**